.github/workflows/snyk.yml

```diff
@@ -0,0 +1,16 @@
+name: Snyk
+on:
+    schedule:
+        - cron: '0 6 * * *'
+    push:
+        branches:
+            - main
+    pull_request:
```

Comment:
Should this be running on pull_request if the above action already is?

Reply:
Good shout, no.

Reply:
```suggestion
```